readthedocs/builds/managers.py

```diff
@@ -179,3 +179,37 @@ class InternalBuildManager(SettingsOverrideObject):
 
 class ExternalBuildManager(SettingsOverrideObject):
     _default_class = ExternalBuildManagerBase
+
+
+class VersionAutomationRuleManager(PolymorphicManager):
+
+    def append_rule(
```

Comment:
nit: I'd use `add_rule` to keep consistency with other methods that we already have, like `add_subproject`.

Reply:
This is part of #5995